### ISSUE TITLE
Building cf-staticfile-buildpack-private-s3

### ISSUE THREAD

**Issue by sergioRozenszajn** [amaline/cf-staticfile-buildpack-private-s3#1]
Hi Al,

I'm trying to build a staticfile buildpack using Deploy.sh script without success:
fatal: unable to access 'https://@/': Received HTTP code 400 from proxy after CONNECT

Can you provide a usage example?

Thanks, Sergio

**Comment by amaline** [comment]
The circle CI project needs to have a GITHUB_TOKEN environmental variable set in order to push the build back to github.  Not really maintaining this, it was just a proof of concept.

**Comment by amaline** [comment]
Note that this custom buildpack will pull in https://github.com/amaline/s3-proxy-nginx/releases/download/v0.6/nginx-s3.tgz as the nginx server.  There is a newer version of the s3 signing module available: https://github.com/anomalizer/ngx_aws_auth

**Comment by sergioRozenszajn** [comment]
Thank you. Actually we are trying to use Cloud Foundry Static file buildpack, which uses NGINX. The problem is that the installed NGINX does not support s3 signing module. So we forked Staticfile buildpack and pointed to https://github.com/amaline/s3-proxy-nginx/releases/download/v0.6/nginx-s3.tgz as a trial. Unfortunately we did not manage to generate the V4 signature correctly in NGINX.conf (after performing all steps as per AWS documentation) --> it complains that signature is not correct.
So I'm trying to find a good example

**Comment by amaline** [comment]
Did you follow the documentation found in https://github.com/anomalizer/ngx_aws_auth ? (although maybe using the https S3 endpoints).  I searched for, but could not find the configuration I got working.  But I think I just followed the examples in the anomalizer repository.  I did just run a new build switching back to the current anomalizer/ngx_aws_auth module instead of my fork since I think all issues that caused me to fork have been resolved: https://github.com/amaline/s3-proxy-nginx/releases/download/v0.8.7/nginx-s3.tgz
